lisa/sut_orchestrator/azure/platform_.py

```diff
@@ -2457,6 +2472,10 @@ def _add_image_features(self, node_space: schema.NodeSpace) -> None:
 
             generation = _get_vhd_generation(image_info)
             node_space.features.add(features.VhdGenerationSettings(gen=generation))
+        if azure_runbook.vhd or azure_runbook.shared_gallery:
```

Comment:
use `elif` or `else`. The logic is related to runbook type. `elif` and `else` helps understand they are related.

Reply:
updated